docs/_templates/layout.html

```diff
@@ -5,7 +5,6 @@
     <link rel="apple-touch-icon" href="{{ pathto('_static/' ~ theme_touch_icon, 1) }}" />
   {% endif %}
   <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9">
-  <script type="text/javascript" src="https://gumroad.com/js/gumroad.js"></script>
```

Comment:
Why? Because this line is a duplicate of https://github.com/kennethreitz/requests/pull/2915/files#diff-cac3dc3bfd42bc240bc9eb924f8dbfddR18.